src/mobile/src/screens/HomeScreen.jsx

```diff
@@ -1,9 +1,21 @@
-import { View, Text } from 'react-native';
+import { View, StyleSheet } from 'react-native';
+import Banner from '../components/Banner/Banner';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    marginTop: 30,
+  },
+});
+
+const HomeScreen = ({ navigation }) => {
+  const navigateToContact = () => {
+    navigation.navigate('Contact');
+  };
 
-const HomeScreen = () => {
   return (
-    <View>
-      <Text>Home Screen</Text>
+    <View style={styles.container}>
```

Comment:
Could you use Safeview? because the top part is too close to the notification bar, It is easy to on the notification bar instead of the top button.https://reactnative.dev/docs/safeareaview

From inspect tool
![image](https://user-images.githubusercontent.com/58233223/155203869-ba6c5f11-3b2a-49f2-9c3d-8934727a7f3b.png)

Reply:
@Kevan-Y  The safeviewarea just did not work for me. Thats the reason why I chose to use margin top instead.
 You can see here: https://stackoverflow.com/questions/59428494/safeareaview-not-work-views-go-above-the-notification-tab
Also May be I'll try safeviewarea from 'react-native-safe-area-context' to see if it works.

Reply:
@Kevan-Y the SafeAreaView from 'react-native-safe-area-context' works for me.